dmutils/content_loader.py

```diff
@@ -494,6 +497,10 @@ def _optional_form_fields(self):
         else:
             return []
 
+    def inject_brief_questions_into_boolean_list_question(self, brief):
```

Comment:
Also here - maybe not mention `brief` in the name. And do we really want the method name to be exactly the same? I find it confusing having two methods wit the same name in the same file, even if they do belong to different classes.